Bottom: 4ee43da9860beddaca1dc45070b9a7506f90605b
Top:    e5dd63740734ad3a6b998e535d168051eb4078cf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-13 02:10:00 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!






---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index b09d948..93b6073 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -310,13 +310,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
